lib/phantomjs/core.js

```diff
@@ -328,12 +280,12 @@ function getCriticalPathCss (options) {
   debuglog('stripped out non matching media queries')
 
   page.open(options.url, function (status) {
-    if (status !== 'success') {
```

Comment:
I just checked the context and given how short the function is it doesn't matter to me. I'll merge as is. Otherwise what I meant was:
```
if (status !== 'success') {
  errorlog('Error opening url \'' + page.reason_url + '\': ' + page.reason) // jshint ignore: line
  phantomExit(1)
  return
}
debuglog('page opened')
page.evaluate(pruneNonCriticalCss, astRules, options.forceInclude, GENERATION_DONE)
```